specifyweb/frontend/js_src/lib/components/FormSliders/IntegratedRecordSelector.tsx

```diff
@@ -240,7 +241,8 @@ export function IntegratedRecordSelector({
                 preHeaderButtons={collapsibleButton}
                 sortField={sortField}
                 viewName={viewName}
-                onAdd={(): void => {
+                onAdd={(resources): void => {
+                  if (!isInteraction) collection.add(resources);
```

Comment:
can you please double-check if this line should be added here, or on line 122
(if you add it on line 122, this could would be inside the handleAdd() function, and thus also affect the  line 184)

Reply:
Yes I do need it there.

Reply:
I understand that you need it here
I am saying that if you would move this code, than it would apply in both places where handleAdd is called
can you check if this code is needed only here or in both places where handleAdd is called?